src/tests/multihost/ad/test_access_control.py

```diff
@@ -0,0 +1,409 @@
+            2. Any AD-User log in should be denied
+        """
+        tools = sssdTools(multihost.client[0])
+        (aduser, _) = create_aduser_group
+        domain_name = tools.get_domain_section_name()
+        dom_section = f'domain/{domain_name}'
+        sssd_params = {'access_provider': 'simple',
+                       'simple_allow_users': '$'}
+        tools.sssd_conf(dom_section, sssd_params, action='add')
+        tools.clear_sssd_cache()
+        _, stdout = ssh_login(multihost, f'{aduser}@{domain_name}')
+        assert stdout, 'Access denied log is absent in /var/log/secure'
+
+    @staticmethod
+    def test_simple_allow_user_to_invalid_user(multihost, create_aduser_group, backupsssdconf):
+        """
+        :title: Set simple_allow_user to an invalid user
+        :id: fdde80ab-1364-4953-893e-62f7c3377ff8
+        :description: Setting simple_allow_user to an invalid user, with this sssd would deny log in of all users
+        :steps:
+            1. Set simple_allow_user to non existing user in sssd.conf and restart sssd
+            2. Log in as AD user
+        :expectedresults:
+            1. Changes should reflected in restarted sssd
+            2. User log in should be denied
+        """
+        tools = sssdTools(multihost.client[0])
+        (aduser, _) = create_aduser_group
+        domain_name = tools.get_domain_section_name()
+        dom_section = f'domain/{domain_name}'
+        sssd_params = {'access_provider': 'simple',
+                       'simple_allow_users': f'nonuser@{domain_name}'}
+        tools.sssd_conf(dom_section, sssd_params, action='add')
+        tools.clear_sssd_cache()
+        _, stdout = ssh_login(multihost, f'{aduser}@{domain_name}')
+        assert stdout, 'Access denied log is absent in /var/log/secure'
+
+    @staticmethod
+    def test_simple_deny_user_to_user1(multihost, create_aduser_group, backupsssdconf):
+        """
+        :title: Set simple_deny_user to a user
+        :id: 8a068411-4ac7-4b15-b21e-210f3473d164
+        :description: Setting simple_deny_user to an AD-user, with this sssd would deny log in of that user only
+        :steps:
+            1. Set simple_deny_user to a user in sssd.conf and restart sssd
+            2. Log in as AD user
+        :expectedresults:
+            1. SSSD should load changes and start successfully
+            2. User log in should be denied
+        """
+        tools = sssdTools(multihost.client[0])
+        (aduser, _) = create_aduser_group
+        domain_name = tools.get_domain_section_name()
+        dom_section = f'domain/{domain_name}'
+        sssd_params = {'access_provider': 'simple',
+                       'simple_deny_users': f'{aduser}@{domain_name}'}
+        tools.sssd_conf(dom_section, sssd_params, action='add')
+        tools.clear_sssd_cache()
+        _, stdout = ssh_login(multihost, f'{aduser}@{domain_name}')
+        assert stdout, 'Access denied log is absent in /var/log/secure'
```

Comment:
Hi,

I would suggest to add a test with a different user to show that only `{aduser}@{domain_name}` is rejected.

bye,
Sumit